### ISSUE TITLE
Mouse and mini-joystick mode

### ISSUE THREAD

**Issue by antegorov** [XITRIX/Moonlight-Switch#163]
Good afternoon! Now, if you go into mouse use mode, moving the cursor is the left mini-joystick, and scrolling is the right one. Tell me, can you add a function to select which stick to use for each function? Or should I swap these places separately?
I would like the cursor to be moved on the right mini-joystick, and scrolling to be on the left. Thank you.

**Comment by XITRIX** [comment]
Yes, I can add setting to swap left and right sticks functionality for that
